src/bmp_reader.cpp

```diff
@@ -1,31 +1,25 @@
 #include "bmp_reader.h"
 #include <iostream>
 #include <algorithm>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
 
 bool ReadBMP(std::string imagepath, unsigned char *&header, unsigned char *&rgbData, unsigned int &headerSize, unsigned int &imageSize)
 {
-    // BMP Header
-    header = new unsigned char[HEADER_SIZE];
-    unsigned int width;
-    unsigned int height;
+    FILE *file;
```

Comment:
Why did you rewrite the ReadBMP function?

Reply:
Porque Replit no me estaba leyendo las bmp y decidí reecribir la función para ver si el editor me las ejecutaba